index.bs

```diff
@@ -28,6 +28,72 @@ Blah blah blah.
 
 This specification depends on the Infra standard. [[!INFRA]]
 
+<h2 id="deployed-mitigations">Deployed Mitigations</h2>
+
+Some browsers have deployed and announced protections against
+navigational-tracking. This section is a work in progress to detail what
+protections have been shipped and / or are planned. This section should not
+be understood to be comprehensive.
+
+<h3>Safari</h3>
+Safari uses an algorithmic approach to combat navigational-tracking. Safari
+labels a site as having navigational-based cross-site tracking capability if
+the following criteria are met:
+
+<ul>
+  <li>The site has not served content as a third-party.</li>
+  <li>The site automatically redirects the user to another site, immediately
+      or after a short delay.</li>
+  <li>The site has not received a user activation.</li>
+</ul>
+
+Sites can be "tainted" as having cross-site tracking capabilities if they
+redirect to sites already classified as having cross-site tracking capabilities.
+For example, consider the case of a user clicking on a link on
+<code>start.example</code>, which redirects to <code>second.example</code>,
+which redirects to <code>third.example</code>, which redirects to
+<code>end.example</code>. If Safari has classified <code>third.example</code> as
+having tracking capabilities, the above behavior may result in Safari classifying
+<code>second.example</code> as having cross-site tracking capabilities.
+
+Safari deploys defenses against navigational-tracking if a navigation matches
+all of the following criteria:
+
+<ul>
+  <li>The site (determined by eTLD+1) of the user is navigating <em>from</em>
+      has been classified as having cross-site tracking capabilities</li>
+  <li>The URL the user is navigating <em>to</em> has either query parameters
+      or a URL fragment</li>
+</ul>
+
+When the above criteria holds the following defenses are deployed:
+
+<ul>
+  <li>The lifetime of client-side set cookies on the <em>destination page</em>
+      is capped at 24 hours.</li>
+  <li>If the user does not interact (i.e., register a user activation) on the
+      site within seven days of browser use, all non-cookie storage is data is
+      deleted.</li>
+</ul>
+
+<h3>Firefox</h3>
+Firefox uses a list-based approach to combat navigational-tracking. All storage
+for list-labeled sites is cleared after 24 hours, unless the user has interacted
+with the site in the first-party context in the last 45 days. Firefox uses
+the Disconnect list to determine which sites are tracking sites.
```

Comment:
I might say 

```suggestion
Firefox uses a list-based approach to combat navigational-tracking. All storage
for tracking sites is cleared after 24 hours, unless the user has interacted
with the site in the first-party context in the last 45 days. Firefox uses
the Disconnect list to determine which sites are tracking sites.
```
since you clarify that "tracking" sites are those on the Disconnect list later in the paragraph. You could also swap these two sentences, so

```suggestion
Firefox uses a list-based approach to combat navigational-tracking. Sites on the
Disconnect list are considered tracking sites. All storage
for tracking sites is cleared after 24 hours, unless the user has interacted
with the site in the first-party context in the last 45 days.
```